### PR TITLE
adding webvr eventcompat data

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/mdn/sprints/issues/948.

As well as adding compat data for the seven events listed here, I also added compat data for

vrdisplaypointerrestricted event
onvrdisplaypointerrestricted
vrdisplaypointerunrestricted event
onvrdisplaypointerunrestricted

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -7452,6 +7554,545 @@
           }
         }
       },
+      "vrdisplayactivate": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplayactivate_event": {
```

File: api/Window.json [hunk 2]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplayblur": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplayblur_event": {
```

File: api/Window.json [hunk 3]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplayconnect": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplayconnect_event": {
```

File: api/Window.json [hunk 4]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplaydeactivate": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplaydeactivate_event": {
```

File: api/Window.json [hunk 5]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplaydisconnect": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplaydisconnect_event": {
```

File: api/Window.json [hunk 6]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplayfocus": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplayfocus_event": {
```

File: api/Window.json [hunk 7]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplaypresentchange": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplaypresentchange_event": {
```

File: api/Window.json [hunk 8]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplaypointerrestricted": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplaypointerrestricted_event": {
```

File: api/Window.json [hunk 9]
```diff
@@ -7452,6 +7554,545 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vrdisplaypointerunrestricted": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "vrdisplaypointerunrestricted_event": {
```

**Comment by chrisdavidmills** [comment]
Thanks @Elchi3 ; fixed!

**Pull request merged**